node/core/candidate-validation/src/tests.rs

```diff
@@ -414,6 +423,7 @@ fn candidate_validation_ok_is_ok() {
 		Arc::new(pov),
 		Duration::from_secs(0),
 		&Default::default(),
+		commitments.hash(),
```

Comment:
I am missing a dedicated test, that makes sure that the commitment hash is now actually checked. It looks like we only recycled other tests and make them fail for a different reason?

Reply:
done